proposals/0002-within-polygon.md

```diff
@@ -0,0 +1,29 @@
+# Feature name
+
+* Proposal: 0002-within-polygon
+* Authors: [Mads Bjerre](https://github.com/madsb)
+* Review Manager: TBD
+* Status: **Awaiting review**
+
+## Introduction
+## Motivation
+
+Parse currently supports query constraints `withinGeoBox`, `withinKilometers`, `withinMiles` and `withinRadians`. Assume we want to find objects within a polygon.
+
+## Proposed solution
+
+Introducing a `withinPolygon` constraint that takes an array of lat/lon coordinates, or even nested arrays of coordinates for MultiPolygon support.
```

Comment:
Can you expand on that and describe with the JSON payloads how it should look like?